docs/architecture/psa-migration/psa-limitations.md

```diff
@@ -104,7 +120,7 @@ algorithms can differ from each other.
     - hash alg used for message hashing, encoding and MGF1
     - salt length can be either "standard" (== hashlen) or "any"
   - signature generation:
-    - salt length: always using the maximum legal value
+    - salt length: always using the maximum legal value and random salt
```

Comment:
salt length: always using the maximum legal value according to FIPS 186 (which is the hash length unless it doesn't fit). PKCS#1 allows longer salts.